packages/synapse-react-client/src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -56,7 +56,7 @@ export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
       const start = textVal?.selectionStart
       const textBeforeTag = text.substring(0, start)
       const textAfterTag = text.substring(start, text.length)
-      setSelectionStart(start + user.length + 1)
+      setSelectionStart(start + user.length)
```

Comment:
Adjust so that after a user tag is inserted in the middle of text, the cursor is positioned just after the user tag, rather than one character further. 

See example of what happened previously -- after the user tag is inserted, the cursor is positioned after 5, but we would expect it to be positioned after `@test`: 

https://user-images.githubusercontent.com/26949006/236282101-be94abe0-0139-4528-a5ae-4055e089e6aa.mov

The updated behavior can be seen in this [comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/187/files#r1185307858).